mobile/src/common/types/navigation/drawer-navigation-list.type.ts

```diff
@@ -0,0 +1,14 @@
+import { AppScreenName } from '~/common/enums/enums';
+
+type DrawerNavigationListSubroutes = {
+  name: AppScreenName;
+  iconName: string;
+};
+
+type DrawerNavigationList = {
+  name: string;
+  subroutes: DrawerNavigationListSubroutes[];
```

Comment:
Your already have DrawerNavigationItem, use it instead
```suggestion
  subroutes: DrawerNavigationItem[];
```